src/Razor/Directory.Build.props

```diff
@@ -11,6 +11,9 @@
     <ProjectReference Include="$(MSBuildThisFileDirectory)..\Analyzers\Razor.Diagnostics.Analyzers\Razor.Diagnostics.Analyzers.csproj"
                       PrivateAssets="all"
                       ReferenceOutputAssembly="false"
+                      IncludeOutputGroupsInVSIX=""
+                      IncludeOutputGroupsInVSIXLocalOnly=""
```

Comment:
It's a bit odd to me that these are empty values and not "false". Why is that?

Reply:
I believe the value is the list of things to include, so empty value means include nothing. E.g. https://github.com/dotnet/project-system/blob/main/setup/VisualStudioEditorsSetup/VisualStudioEditorsSetup.csproj

Reply:
Correct, contrary to the name this isn't a boolean, its a list of groups.

(but also my MSBuild foo extends as far as [copying what other people do](https://github.com/dotnet/roslyn/blob/main/src/VisualStudio/Setup.ServiceHub/x64/Roslyn.VisualStudio.ServiceHub.Setup.x64.csproj#L16) and accepting that as good enough 😁)

Reply:
Ah ha! Thank you for the explaination.